src/Admin/AbstractAdmin.php

```diff
@@ -3162,6 +3164,11 @@ public function getDashboardActions()
      */
     final public function isDefaultFilter($name)
     {
+        @trigger_error(sprintf(
+            'Method "%s" is deprecated since sonata-project/admin-bundle 3.73.',
```

Comment:
Prefer
```suggestion
            'Method "%s" is deprecated since sonata-project/admin-bundle 3.x.',
```

We generally replace all 3.x with the right version number.
This will avoid you to update the message if we release a new version before the merge ;)